src/burnchains/burnchain.rs

```diff
@@ -723,7 +756,7 @@ impl Burnchain {
                 let ipc_block = downloader.download(&ipc_header)?;
                 let download_end = get_epoch_time_ms();
 
-                debug!("Downloaded block {} in {}ms", ipc_block.height(), download_end.saturating_sub(download_start));
+                debug!("Downloaded block {} in {}ms", ipc_block.height(), download_end - download_start);
```

Comment:
Please revert this to using a saturating subtraction.  This has lead to runtime panics in the past, as reported by @psq.

Reply:
Ah, oops -- I copied this "deprecated indexer path" from a commit tag that didn't have that fix in place.

Reply:
Fixed in be0f72ac8